src/SprykerEco/Zed/Payolution/Communication/Plugin/Oms/Command/AbstractPayolutionCommandPlugin.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payolution\Communication\Plugin\Oms\Command;
+
+use Orm\Zed\Sales\Persistence\SpySalesOrder;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+
+abstract class AbstractPayolutionCommandPlugin extends AbstractPlugin
+{
+    /**
+     * @param array $orderItems
+     * @param \Orm\Zed\Sales\Persistence\SpySalesOrder $orderEntity
+     *
+     * @return array
+     */
+    protected function getPartialOrderItems(array $orderItems, SpySalesOrder $orderEntity): array
+    {
+        if (count($orderItems) < count($orderEntity->getItems())) {
```

Comment:
Didn't get why you need such conditions. You should always trust to `$orderItems` parameter as it always represents order items list to be affected by oms command.